.ci/close-create-milestone.sh

```diff
@@ -7,10 +7,10 @@ source ./.ci/util.sh
 checkForVariable "GITHUB_TOKEN"
 
 echo "Close previous milestone at github"
-MILESTONE_ID=$(curl -s \
+MILESTONE_ID=$(curl --fail-with-body -s \
```

Comment:
Is it an issue to do this inside a `$()`?

Reply:
Should not be issue, it is good to fail script if this curl failing